rmgpy/rmg/main.py

```diff
@@ -735,9 +736,14 @@ def execute(self, **kwargs):
                                         simulatorSettings = simulatorSettings,
                                         conditions = self.rmg_memories[index].get_cond()
                                     )
-                                self.updateReactionThresholdAndReactFlags(
+                                    self.updateReactionThresholdAndReactFlags(
```

Comment:
I think here we're capturing exceptions from `simulate()`. Perhaps `updateReactionThresholdAndReactFlags`  should be relocated to an `else` clause after the try-except?